track/__main__.py

```diff
@@ -48,31 +47,49 @@ def utc2localtime(utc):
 
 online = None
 last_offline = None
+
+printName = True
 while True:
-    if contact_id in ['me', 'self']:
+    try:
+        # File IO
+        dataFile = open('trackerLogs.txt', 'a+', encoding = 'utf-8')
+        if contact_id in ['me', 'self']:
         # Workaround for the regression in Telethon that breaks get_entity('me'):
         # https://github.com/LonamiWebs/Telethon/issues/1024
-        contact = client.get_me()
-    else:
-        contact = client.get_entity(contact_id)
+            contact = client.get_me()
+        else:
+            contact = client.get_entity(contact_id)
+            user = contact.first_name+' '+contact.last_name
+        if printName == True:
+            print(user, file = dataFile)
+            print(user)
 
-    if isinstance(contact.status, UserStatusOffline):
-        if online != False:
-            online = False
-            print(f'={utc2localtime(contact.status.was_online).strftime(DATETIME_FORMAT)}: User went offline.')
-        elif last_offline != contact.status.was_online:
-            if last_offline is not None:
-                print(f'={utc2localtime(contact.status.was_online).strftime(DATETIME_FORMAT)}: User went offline after being online for short time.')
-            else:
+        if isinstance(contact.status, UserStatusOffline):
+            if online != False:
+                online = False
+                print(f'={utc2localtime(contact.status.was_online).strftime(DATETIME_FORMAT)}: User went offline.', file = dataFile)
```

Comment:
No spaces should be around `=` that is used to indicate a keyword argument.

https://www.python.org/dev/peps/pep-0008/#other-recommendations